lib/docs/src/checkbox.stories.tsx

```diff
@@ -43,7 +43,12 @@ export const Primary = (props: Props): JSX.Element => {
 };
 
 export const Basic = (): JSX.Element => {
-	return <Checkbox>Default</Checkbox>;
+	const [checked, setChecked] = useState(false);
+	return (
+		<Checkbox checked={checked} setChecked={() => setChecked(!checked)}>
```

Comment:
It's better to use the param provided by setChecked instead of using the "checked" state directly:

```
setChecked={(c) => setChecked(c)}>
```

Because of this you can pass the state updater directly

```
setChecked={setChecked}>
```